### PR TITLE
[anomalies] Add fix for date range for monthly upload frequency.

### PR DESCRIPTION
**Pull request by xor-xor**

### CODE REVIEW

**Review by mkurek — APPROVED** [review]

File: src/ralph_scrooge/management/commands/detect_usage_anomalies.py [hunk 1]
```diff
@@ -102,10 +102,19 @@ def _get_usage_values_for_month(usage_types, end_date):
...
+    An exception to this "end_date - 1 month" rule are usage types with
+    `upload_frequency` set to `monthly` - in that case, the aforementioned
+    range is extended to the beginning of the month (e.g. '2016-12-01' instead
+    of '2016-12-10' for `start_date`.
     """
     results = {}
     start_date = get_negative_month_range(end_date + a_day).next()
     for ut in usage_types:
+        freq_name = UsageTypeUploadFreq.from_id(ut.upload_freq).name
+        if freq_name == 'monthly':
```
Comment (szok) [inline src/ralph_scrooge/management/commands/detect_usage_anomalies.py]: freq = UsageTypeUploadFreq.from_id(ut.upload_freq).id
if freq  == UsageTypeUploadFreq.monthly.id:
...
?

Comment (xor-xor) [reply src/ralph_scrooge/management/commands/detect_usage_anomalies.py]: So why not just `if ut.upload_freq == UsageTypeUploadFreq.monthly.id`..? :wink:

Comment (szok) [reply src/ralph_scrooge/management/commands/detect_usage_anomalies.py]: +1

Comment (xor-xor) [reply src/ralph_scrooge/management/commands/detect_usage_anomalies.py]: Done.

**Review by szok — APPROVED** [review]

**Pull request merged**